docs/install.md

```diff
@@ -91,12 +91,11 @@ for more information.
    oc login -u system:admin
    ```
 
-1. Set up the project and name space:
+1. Set up the namespace (project) and configure the service account:
 
    ```bash
    oc new-project tekton-pipelines
    oc adm policy add-scc-to-user anyuid -z tekton-pipelines-controller
-   oc apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.notags.yaml
```

Comment:
I think it's still ok to have it here. It's a snippets, and it more clear that the `notags.yaml` should be used :angel:

Reply:
Isn't this more confusing? The command is right below. The command is right below. I had to check very carefully if any character was different.

Reply:
oh lol.. it seems I can't read properly on a friday… nevermind :sweat_smile: